### DISCUSSION TITLE
为什么Menu组件支持getPopupContainer属性，但是api文档中没有列出这个属性呢？

### DISCUSSION THREAD

**Discussion by wh1246935943** [ant-design/ant-design#34257] [Q&A]
为什么Menu组件支持getPopupContainer属性，但是api文档中没有列出这个属性呢？

抱着试一试的心态，奇迹发生了，这个属性在Menu上是可用的！！！

**Comment by yoyo837** [comment]
除了意外没写，就是刻意不暴露不希望开发者使用的，可能发生变更或者删除。

**Comment by afc163** [comment]
可以来补上
